src/components/cylc/graph/Graph.vue

```diff
@@ -0,0 +1,631 @@
+<template>
+  <div class="c-graph">
+    <SyncLoader
+      :loading='loading'
+      :color='vueSpinner.color'
+      :size='vueSpinner.size'
+      class='spinner'
+    ></SyncLoader>
+    <div class='switch-layout ma-0'>
+      <v-expansion-panels
```

Comment:
Using an expansion panel, I think we have more space to display the graph. It's hidden by default, user can click on the expand button to visualize the settings now.